docs/getting-started/faq.md

```diff
@@ -77,7 +96,7 @@ Please follow the instructions [here](../how-to-guides/adding-support-for-a-new-
 
 ### Can the same storage engine be used for both the offline and online store?
 
-Yes. For example, the Postgres [connector](https://github.com/nossrannug/feast-postgres) can be used as both an offline and online store.
+Yes. For example, the Postgres connector can be used as both an offline and online store.
```

Comment:
Wanna point out that postgres can also be used to store the registry?

Reply:
done, though doesn't look like we have any docs about that.